### ISSUE TITLE
Check whether performance of the probe keeps improving even if the pre-training is overfitting

### ISSUE THREAD

**Issue by robertdvdk** [jwpartyka/uva-fomo#18]
We observe that the pre-training is overfitting, which is why we propose to add dropout. However, it might be the case that the network's representations are still improving. We should check whether the fine-tuning performance keeps improving even if the pre-training is overfitting.
